api/client/service/update.go

```diff
@@ -39,6 +39,7 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	addServiceFlags(cmd, opts)
 
 	flags.Var(newListOptsVar(), flagEnvRemove, "Remove an environment variable")
+	flags.Var(newListOptsVar(), flagGroupRemove, "Remove additional user groups from the container")
```

Comment:
Thanks @cpuguy83. The pull request has been updated with the help output `"Remove previously added user groups from the container"` for now. Let me know if there are other suggestions.